sympy/core/evalf.py

```diff
@@ -1391,6 +1391,32 @@ def evalf(self, n=15, subs=None, maxn=100, chop=False, strict=False, quad=None,
             verbose=<bool>
                 Print debug information (default=False)
 
+        Note:
+        It is always suggested to use expr.evalf(subs=...) over,
+        expr.subs(...).evalf() or expr.evalf().subs(...) because of the
+        following reasons:-
+
+        i.   expr.evalf(subs=...) avoids the loss of significance caused due to
+             naive substitution;
+        ii.  expr.evalf(subs=...) the expression is run through the evalf algorithm,
+             which takes into account various issues that can lead to loss of
+             significance.
+        iii. The subs dictionary tells the evalf algorithms which symbols should be
+             replaced with numbers when they are encountered.
+
+        Example
```

Comment:
pluralize, change dashes to "========" and leave a blank line after the underlining.

Reply:
I have made the suggested changes.